calendars/all.ical

```diff
@@ -19,186 +19,6 @@ LOCATION:Denver\, Colorado
 NAME:Supercomputing 2023
 END:VEVENT
 BEGIN:VEVENT
-SUMMARY:Container Plumbing 2023 in virtual hosted by RedHat

```

Comment:
Why are you deleting container plumbing too?

Reply:
That was last week, right?

Reply:
Yeah but why delete it?

Reply:
Hmm, is there any value in keeping past events? What will prevent accumulation of these? (It's not an issue unless we are going to depend on hand-editing to fix corrections, where it will get tedious to wade through old ones getting to current entries.)

Reply:
Yeah that’s fine, I guess it’s nice to have a history but I’m fairly indifferent. I don’t think a lot of folks are using this.